docs/whats_new/v0.8.0.md

```diff
@@ -0,0 +1,120 @@
+# Version 0.8.0
+We're happy to announce the AutoGluon 0.8 release. [Highlights of new features]
+
+As always, only load previously trained models using the same version of AutoGluon that they were originally trained on. Loading models trained in different versions of AutoGluon is not supported.
+
+This release contains XXX commits from XX contributors!
+
+See the full commit change-log here: XXX
+
+Full Contributor List (ordered by # of commits):
+
+XXX
+
+AutoGluon 0.8 supports Python versions 3.8, 3.9, and 3.10.
+
+# Changes
+XXX
+
+## General
+* General doc improvements @tonyhoo @Innixma @yinweisu @gidler @cnpgs @isunli @giswqs (#2940, #2953, #2963, #3007, #3027, #3059, #3068, #3083, #3128, #3129, #3130, #3147, #3174, #3187, #3256, #3258)
+* General code fixes and improvements @yinweisu @Innixma (#2921, #3078, #3113, #3140, #3206)
+* CI improvements @yinweisu @gidler @yzhliu @liangfu @gradientsky (#2965, #3008, #3013, #3020, #3046, #3053, #3108, #3135, #3159, #3283, #3185)
+* New AutoGluon Webpage @gidler @shchur (#2924)
+* Support sample_weight in RMSE @jjaeyeon (#3052)
+* Move AG search space to common @yinweisu (#3192)
+* Deprecation utils @yinweisu (#3206, #3209)
+* Update namespace packages for PEP420 compatibility @gradientsky (#3228)
+
+
+AutoMM
```

Comment:
We probably need to change the formatting here to make sure that it's consistent across modules

Reply:
Yes. Thanks for pointing it out